buildkitd/entrypoint.sh

```diff
@@ -34,6 +34,23 @@ if [ -n "$GIT_URL_INSTEAD_OF" ]; then
 
 fi
 
+# setup git credentials and config
+i=0
+while true
+do
+    varname=GIT_CREDENTIALS_"$i"
+    eval data=\$$varname
+    if [ -n "$data" ]
+    then
+        echo echo \$$varname \| base64 -d > /usr/bin/git_credentials_"$i"
+        chmod +x /usr/bin/git_credentials_"$i"
+    else
+        break
+    fi
+    i=$((i+1))
+done
+echo $GIT_CONFIG | base64 -d > /root/.gitconfig
```

Comment:
```suggestion
echo "$GIT_CONFIG" | base64 -d > /root/.gitconfig
```

Nit - It doesn't really matter here, more like a style comment.